src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala

```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```

Comment:
Ok, then I'll add a new pattern that works they way I want, so you can pick and choose.